eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -136,12 +136,14 @@ jobs:
   - bash: |
       # Run clang-format recursively on each source and header file within the repo sdk folder.
       echo Check clang-formatting
-      clang-format --version
-      find sdk/ -regex '.*\.\(cpp\|hpp\)' -exec clang-format -style=file -i {} \;
+      sudo apt-get update
+      sudo apt-get install clang-format-11
```

Comment:
Just to confirm, is 11 the latest? From what I gather it is:
https://releases.llvm.org/download.html

Reply:
11 is not available for ubuntu 18 by default.  We need to use 10.

Reply:
Then can we use ubuntu 20 for this pipeline?

Reply:
That would be a bigger change. We don't have a ubuntu 20 pipeline atm.

Reply:
Can we use macOS? We can get clang-format 11 easily on macOS.

Reply:
Or how about just downloading binary from this page https://releases.llvm.org/download.html#11.0.0 and extract from the tarball?

Reply:
Turns out there are binaries for Ubuntu 12 and 20 but not for 18  ...  We would need to get source and build which could take some extra time.

Instead, I check with macOS as suggested by @Jinming-Hu  and we get version 11 by default there. So I moved the CI check to be macOS only

Reply:
Yeah, moving to macOS is a good idea. Also, downloading binary is also an option.